build.gradle

```diff
@@ -1,7 +1,6 @@
 plugins {
     // Apply the java-library plugin to add support for Java Library
     id 'java-library'
-    id 'java'
```

Comment:
ahhh good call, can you make sure you clean this up in the service + kiota libs too please?